Panic when converting []uint16 to boolean

I see the following panic from Otto:

```
panic: toBoolean([]uint16) [recovered]
    panic: toBoolean([]uint16)

goroutine 34 [running]:
panic(0x985a00, 0xc4253e30c0)
    /opt/go/src/runtime/panic.go:500 +0x1a1
test/vendor/github.com/robertkrimen/otto.catchPanic.func1(0xc421f67990)
    /opt/gopath/src/test/vendor/github.com/robertkrimen/otto/error.go:247 +0x3ec
panic(0x985a00, 0xc4253e30c0)
    /opt/go/src/runtime/panic.go:458 +0x243
test/vendor/github.com/robertkrimen/otto.Value.bool(0x3, 0x95aa40, 0xc4255db5a0, 0x3)
    /opt/gopath/src/test/vendor/github.com/robertkrimen/otto/value_boolean.go:39 +0x19f
test/vendor/github.com/robertkrimen/otto.(*_runtime).cmpl_evaluate_nodeBinaryExpression(0xc42034a000, 0xc421698300, 0x2, 0xc421f4b8f8, 0x421e51)
    /opt/gopath/src/test/vendor/github.com/robertkrimen/otto/cmpl_evaluate_expression.go:151 +0xbe
test/vendor/github.com/robertkrimen/otto.(*_runtime).cmpl_evaluate_nodeExpression(0xc42034a000, 0xdd09e0, 0xc421698300, 0x0, 0x0, 0xc421f4bcd8)
    /opt/gopath/src/test/vendor/github.com/robertkrimen/otto/cmpl_evaluate_expression.go:37 +0xd42
test/vendor/github.com/robertkrimen/otto.(*_runtime).cmpl_evaluate_nodeSequenceExpression(0xc42034a000, 0xc421696540, 0x1, 0x0, 0x7)
    /opt/gopath/src/test/vendor/github.com/robertkrimen/otto/cmpl_evaluate_expression.go:345 +0x69
test/vendor/github.com/robertkrimen/otto.(*_runtime).cmpl_evaluate_nodeExpression(0xc42034a000, 0xdd0f60, 0xc421696540, 0xc421f4bbc0, 0x2, 0x0)
```

From looking at value_string.go, it looks like value_boolean.go should be handling []uint16 as a UTF-16 string and returning true for length > 0, false otherwise.

--- a/value_boolean.go
+++ b/value_boolean.go
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"math"
 	"reflect"
+	"unicode/utf16"
 )
 
 func (value Value) bool() bool {
@@ -32,6 +33,8 @@ func (value Value) bool() bool {
 		return true
 	case string:
 		return 0 != len(value)
+	case []uint16:
+		return 0 != len(utf16.Decode(value))
 	}
 	if value.IsObject() {
 		return true

--- a/value_test.go
+++ b/value_test.go
@@ -98,6 +98,8 @@ func TestToBoolean(t *testing.T) {
 		//is(toValue(newObject()), true)
 		is(UndefinedValue(), false)
 		is(NullValue(), false)
+		is([]uint16{}, false)
+		is([]uint16{0x68, 0x65, 0x6c, 0x6c, 0x6f}, true)
 	})
 }
 

